plugins/coverimage.koplugin/main.lua

```diff
@@ -259,7 +273,41 @@ function CoverImage:addToMainMenu(menu_items)
                 end,
                 sub_item_table = {
                     {
-                        text = _("Scale, black background"),
+                        text_func = function()
+                            return T(_("Aspect ratio stretch threshold (%1%)"), self.cover_image_stretch_limit )
+                        end,
+                        help_text_func = function()
+                            return T(_("If the image and the screen have a similar aspect ratio (±%1%), stretch the image instead of honoring its aspect ratio."), self.cover_image_stretch_limit )
```

Comment:
```suggestion
                            return T(_("If the image and the screen have a similar aspect ratio (±%1%), stretch the image instead of keeping its aspect ratio."), self.cover_image_stretch_limit )
```

Reply:
That's mine, I'll readily accept the fact that I'm very fond of using 'honor' in this context ;p.

Reply:
You do love to sprinkle the word honor around everywhere. ^_^

Reply:
Honor to whom honor is due. ;)))))